server/routes/search/searchController.test.ts

```diff
@@ -151,5 +158,21 @@ describe('Search controller', () => {
         }),
       )
     })
+    it('should audit details viewed', async () => {
+      req.body = { searchType: 'name', lastName: 'WILSON' }
+      req.session.searchParams = {}
+      req.session.prisonerSearchForm = { searchType: 'name', firstName: 'GOLDIE', lastName: 'WILSON' }
+      req.query = { filters: 'female' }
+      req.id = 'COR_ID'
+      historicalPrisonerService.findPrisonersByName.mockResolvedValue(results)
+      await controller.getSearch(req, res)
+
+      expect(auditService.logPageView).toHaveBeenCalledWith('SEARCH_RESULTS', {
+        who: 'user',
+        subjectId:
+          '{"prisonerSearchForm":{"searchType":"name","firstName":"GOLDIE","lastName":"WILSON"},"filters":["female"]}',
```

Comment:
would be better to log the prison numbers of the people that have been returned in the page of results

Reply:
good idea